source/conf.py

```diff
@@ -27,6 +31,10 @@
 #
 import qiime2
 
+# Required to import from the utils module.
+sys.path.append(".")
+
+from utils import generate_plugin_rst, cleanup_plugin_rst  # noqa: E402
```

Comment:
Can you insert the _absolute path_ into the `sys.path`, and please put it in the first position of the path. Also, please recycle the `root` variable defined below (you can move it up here in with all the import path jazz).